src/components/MainMemo/MemoTextArea/index.tsx

```diff
@@ -0,0 +1,17 @@
+import * as S from './style';
+import * as T from './type';
+
+function AddTextArea({ text, handleClick, border }: T.MemoTextArea) {
+  const onClick = () => {
+    handleClick();
+  };
+  return (
+    <S.TextArea
+      border={border}
+      placeholder={text}
+      onClick={onClick}
+    ></S.TextArea>
```

Comment:
```suggestion
  return (
    <S.TextArea
      border={border}
      placeholder={text}
      onClick={handleClick}
    />
```
`onClick` 함수가 `handleClick` 함수 실행하는 역할만 하는 거라면 굳이 따로 선언하지 않고 바로 쓰면 될 것 같네요~

그리고 이거는 나중에 eslint나 prettier 설정 추가하면 될 것 같은데 안에 자식 컴포넌트 없으면 self closing tag로 써주세요 :)